--- /src/TypeCollection.php	2023-01-27 15:17:07
+++ /src/TypeCollection.php	2023-01-27 15:49:20
@@ -52,7 +52,7 @@
     /**
      * {@inheritDoc}
      */
-    public function current()
+    public function current(): mixed
     {
         return $this->collection[$this->position];
     }
@@ -60,7 +60,7 @@
     /**
      * {@inheritDoc}
      */
-    public function key()
+    public function key(): mixed
     {
         return $this->position;
     }
@@ -84,7 +84,7 @@
     /**
      * {@inheritDoc}
      */
-    public function valid()
+    public function valid(): bool
     {
         return isset($this->collection[$this->position]);
     }
--- /src/Type.php	2023-01-27 15:24:04
+++ /src/Type.php	2023-01-27 15:47:09
@@ -82,7 +82,7 @@
         if (isset($data['enum'])) {
             $class->setEnum($data['enum']);
         }
-        if (\mb_substr($name, -1) === '?') {
+        if (null !== $name && \mb_substr($name, -1) === '?') {
             $class->setRequired(false);
             $class->setName(\mb_substr($name, 0, -1));
         }
--- /src/SecurityScheme/SecuritySettings/DefaultSecuritySettings.php	2023-01-27 15:16:38
+++ /src/SecurityScheme/SecuritySettings/DefaultSecuritySettings.php	2023-01-27 15:23:24
@@ -70,7 +70,7 @@
      *
      * @return bool
      */
-    public function offsetExists($offset)
+    public function offsetExists($offset): bool
     {
         return isset($this->settings[$offset]);
     }
@@ -88,7 +88,7 @@
      * Get a single settings value
      * @link http://php.net/manual/en/arrayaccess.offsetget.php
      */
-    public function offsetGet($offset)
+    public function offsetGet($offset): mixed
     {
         return $this->settings[$offset] ?? null;
     }
--- /src/TraitCollection.php	2023-01-27 15:16:46
+++ /src/TraitCollection.php	2023-01-27 15:22:08
@@ -58,7 +58,7 @@
     /**
      * @return TraitDefinition
      */
-    public function current()
+    public function current(): TraitDefinition
     {
         if ($this->valid()) {
             return $this->collection[$this->position];
@@ -70,7 +70,7 @@
     /**
      * @return int
      */
-    public function key()
+    public function key(): int
     {
         return $this->position;
     }
@@ -88,7 +88,7 @@
     /**
      * @return bool
      */
-    public function valid()
+    public function valid(): bool
     {
         return isset($this->collection[$this->position]);
     }
--- /src/Body.php	2023-01-27 15:49:58
+++ /src/Body.php	2023-01-27 15:51:03
@@ -228,7 +228,7 @@
      */
     public function getExample()
     {
-        return $this->examples[0];
+        return $this->examples[0] ?? '';
     }
 
     /**
--- /src/Parser.php	2023-01-27 15:47:44
+++ /src/Parser.php	2023-01-27 15:48:50
@@ -693,7 +693,7 @@
             return $this->loadAndParseFile($structure->getValue(), $rootDir);
         }
 
-        if (\mb_strpos($structure, '!include') === 0) {
+        if (null !== $structure && \mb_strpos($structure, '!include') === 0) {
             return $this->loadAndParseFile(\str_replace('!include ', '', $structure), $rootDir);
         }
 
